src/pycmor/std_lib/__init__.py

```diff
@@ -292,6 +296,110 @@ def set_variable_attributes(data: Union[DataArray, Dataset], rule: Rule) -> Unio
     return set_variable_attrs(data, rule)
 
 
+def set_coordinate_attributes(data: Union[DataArray, Dataset], rule: Rule) -> Union[DataArray, Dataset]:
+    """
+    Set CF-compliant metadata attributes on coordinate variables.
+
+    This function applies standardized CF attributes (standard_name, axis,
+    units, positive) to coordinate variables (latitude, longitude, vertical
+    coordinates, etc.) to ensure proper interpretation by xarray and other
+    CF-aware tools.
+
+    Time coordinates are handled separately in the file saving step.
+
+    Parameters
+    ----------
+    data : xarray.DataArray or xarray.Dataset
+        The data to which coordinate attributes will be added.
+    rule : Rule
+        The rule containing configuration for coordinate attribute setting.
+
+    Returns
+    -------
+    xarray.DataArray or xarray.Dataset
+        The data with updated coordinate attributes.
+
+    Notes
+    -----
+    This function sets:
+    - standard_name: CF standard name for the coordinate
+    - axis: X, Y, Z, or T designation
+    - units: Physical units (degrees_east, degrees_north, Pa, m, etc.)
+    - positive: Direction for vertical coordinates (up or down)
+    - coordinates: Attribute on data variables listing their coordinates
+
+    Configuration options:
+    - xarray_set_coordinate_attributes: Enable/disable coordinate attrs
+    - xarray_set_coordinates_attribute: Enable/disable 'coordinates' attr
+
+    Examples
+    --------
+    >>> ds = xr.Dataset({
+    ...     'tas': (['time', 'lat', 'lon'], data),
+    ... }, coords={'lat': lats, 'lon': lons})
```

Comment:
I'd format this through `black`:


```suggestion
   >>> ds = xr.Dataset(
   ...    data={
   ...        "tas": (["time", "lat", "lon"], data),
   ...    },
   ...    coords={"lat": lats, "lon": lons}
   ...)
```

By the way, what is `data`, `lats`, and `lons` in this context? Undefined! This probably will not pass the doc-test step 😉